x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/details_page/components/header/right_content.tsx

```diff
@@ -130,36 +132,55 @@ export const HeaderRightContent: React.FunctionComponent<HeaderRightContentProps
-                                  values={{ count: agentPolicy.unprivileged_agents || 0 }}
+                        <EuiFlexGroup direction="row" gutterSize="xs" alignItems="center">
+                          <EuiFlexItem grow={false}>
+                            <EuiToolTip
+                              content={
+                                <EuiFlexGroup direction="column" gutterSize="xs">
+                                  <EuiFlexItem>
+                                    <FormattedMessage
+                                      id="xpack.fleet.policyDetails.summary.usedByUnprivilegedTooltip"
+                                      defaultMessage="{count, plural, one {# unprivileged agent} other {# unprivileged agents}}"
+                                      values={{ count: agentPolicy.unprivileged_agents || 0 }}
+                                    />
+                                  </EuiFlexItem>
+                                  <EuiFlexItem>
+                                    <FormattedMessage
+                                      id="xpack.fleet.policyDetails.summary.usedByPrivilegedTooltip"
+                                      defaultMessage="{count, plural, one {# privileged agent} other {# privileged agents}}"
+                                      values={{
+                                        count:
+                                          (agentPolicy.agents || 0) -
+                                          (agentPolicy.unprivileged_agents || 0),
+                                      }}
+                                    />
+                                  </EuiFlexItem>
+                                </EuiFlexGroup>
+                              }
+                            >
+                              <LinkedAgentCount
+                                count={agentPolicy.agents || 0}
+                                agentPolicyId={agentPolicy.id}
+                                showAgentText={true}
+                              />
+                            </EuiToolTip>
+                          </EuiFlexItem>
+                          {getRootIntegrations(agentPolicy.package_policies || []).length > 0 &&
+                            (agentPolicy.unprivileged_agents || 0) > 0 && (
+                              <EuiFlexItem grow={false}>
+                                <EuiIconTip
+                                  type="warning"
+                                  color="warning"
+                                  content={
+                                    <FormattedMessage
+                                      id="xpack.fleet.policyDetails.summary.containsUnprivilegedAgentsWarning"
+                                      defaultMessage="This agent policy contains integrations that require Elastic Agents to have root privileges. Some enrolled agents are unprivileged."
```

Comment:
```suggestion
                                      defaultMessage="This agent policy contains integrations that require Elastic Agents to have root privileges. Some enrolled agents are running in unprivileged mode."
```
It's a bit longer but I'd suggest the above. Also, maybe we could have a "Learn more..." link? I don't have a page set up yet to link to yet but I can create a placeholder.

Reply:
Oops. I meant to put this as a reply. :-)

Reply:
this content is in a tooltip so I don't think a link inside the copy will work. we could just link the warning icon itself though, but I'll leave that out of this PR. thanks for the copy suggestion :)